docs/Glossary.md

```diff
@@ -13,6 +13,16 @@ description: >
 
 The following terms are arranged in the order of their appearance in the actual user workflow.
 
+### Projects
+**A project is a set of related [Data Scopes](#data-scope), the Data Scopes may contains different [Data Entities](#data-entities) from arbitrary [Data Sources](#data-sources) for Metric Computation**
+
+Project was introduced in v0.15 to solve some complex metric computation problems. For example, There was a **Change Failure Rate** in [DORA Metric](../UserManuals/DORA) which requires connecting Incident(issue.type = 'INCIDENT') to its related Deployment(cicd_task.type = 'DEPLOYMENT') to determine whether a Deployment failed or not. Unfortunately, most data sources do not offer this relationship, the only reasonable approach so far is to connect Incident to its Closest Deployment, without Project, 
```

Comment:
1. "Project was introduced in v0.15" to "Project will be introduced in v0.15"
2. typo: "For example, There was" to "For example, there is"
3. "the only reasonable approach so far is to connect Incident to its Closest Deployment, without Project," to "the only reasonable approach so far is to map an Incident to its closest Deployment."
4. why is there a ", without Project," end with a `,`?

Reply:
Is it expected that Incident issues always have an associated CICD Task?